src/playwright/matchers/to-dispatch-events.ts

```diff
@@ -13,7 +13,7 @@ interface SerializableEvent {
 export default expect.extend({
   async toDispatchEvents(
     locator: Locator,
-    action: () => Promise<void>,
+    action: () => Promise<unknown>,
```

Comment:
Matches the type [here](https://github.com/CrowdStrike/glide-core/blob/main/src/playwright/test.ts#L54) now. We don't care about the return type.